src/main/scala/beam/agentsim/infrastructure/ScaleUpCharging.scala

```diff
@@ -29,42 +32,42 @@ import scala.util.Random
 
 trait ScaleUpCharging extends {
   this: ChargingNetworkManager =>
+
   import ScaleUpCharging._
 
   private lazy val rand: Random = new Random(beamConfig.matsim.modules.global.randomSeed)
```

Comment:
Since this `Random` is used in a multi-threading environment then it's better to use `ThreadLocalRandom` for performance reason.

Reply:
Okay